ietf/doc/tests.py

```diff
@@ -2705,24 +2705,27 @@ def should_404(self, argdict):
         r = self.client.get(url)
         self.assertEqual(r.status_code, 404)
 
+    # This takes a _long_ time (32s on a 2022 m1 macbook pro) - is it worth what it covers?
```

Comment:
Seems like a lot of permutations being checked, maybe cut it down to just the pdf extension? Another option would be to mock out the pdf generation and just check that it calls the expected method.